docker/base/Dockerfile

```diff
@@ -40,10 +40,9 @@ RUN apt-get update && \
         lsb-release \
         net-tools \
         psmisc \
-        python \
-        python-dbg \
-        python-dev \
-        python-pip \
+        python3-dbg \
```

Comment:
Do we still need all of these when we compile our own python3 ?

The previous packages where for python2, which was unsupported/unused anyway.

Reply:
Probably not but I didn't want to take the risk of removing it.
How about we leave a TODO to get rid of these or the building python from source (it probably makes more sense to install python from source for us right?)

Reply:
Having multiple python3 versions probably adds more possibility for subtle issues than not. Let's remove here and I can see if this breaks things.

Reply:
Done.